src/organisms/cards/PolicyCard/PolicyActions.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import accounting from 'accounting';
+import classnames from 'classnames';
+import Text from 'atoms/Text';
+import Layout from 'atoms/Layout';
+import Button from 'atoms/Button';
+import Hide from 'wrappers/Hide';
+import Spacer from 'atoms/Spacer';
+import styles from './policy_card.module.scss';
+
+const ResponsiveText = ({ text, offset = 0 }) =>
```

Comment:
this is super janky but basically bc of the layout, the buttons get super small when changing the width of the browser and this was the only way I could get the font size to respond to the width of the buttons

Reply:
@cisacke I think we should push back on this design for screens between 769-1024. It's not even a matter of these texts, the buttons becoming that small is pretty ridiculous

Reply:
Cool hack. Did you consider making this available as an actual atom.